src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -2880,8 +2880,8 @@ namespace HeatBalanceManager {
                 auto extensibles = objectFields.find("spaces");
                 auto const &extensionSchemaProps = objectSchemaProps["spaces"]["items"]["properties"];
                 if (extensibles != objectFields.end()) {
-                    auto extensiblesArray = extensibles.value();
-                    for (auto extensibleInstance : extensiblesArray) {
```

Comment:
Now let's make a deep copy of each individual element of the array we just made a copy of.